README.md

```diff
@@ -1,15 +1,11 @@
 # Quick Start
 
 ## Prerequisites
-Before running the API you must have an account on the AvN holding a minimum 100 AVT. Set this account's mnemonic or secret seed as the "SURI" environment variable:
-```
-export SURI=<mnemonic OR secret seed>
-```
-examples: \
-`export SURI=caution juice atom organ advance problem want pledge someone senior holiday very` \
-`export SURI=0xc8fa03532fb22ee1f7f6908b9c02b4e72483f0dbd66e4cd456b8f34c6230b849`
+To use the AvN gateway you must hold a minimum 1 AVT in your AvN account. Set this account's mnemonic or secret seed as the "SURI" environment variable, eg: \
+`export SURI=history mule trend shove lawsuit spray fall tongue patient social ribbon tooth` \
```

Comment:
does this work without quotes?
```suggestion
`export SURI="history mule trend shove lawsuit spray fall tongue patient social ribbon tooth"` \
```
When I run it as it is, and run then `echo $SURI` I only get history in the output.